zerver/views/user_groups.py

```diff
@@ -115,6 +119,54 @@ def update_user_group_backend(
     return json_success(request, data)
 
 
+def send_messages_for_user_groups(
```

Comment:
This name could be improved. It's a fairly subtle thing this is doing, so maybe we just need a long name, like `notify_for_user_group_subscription_changes`.